src/AvField.js

```diff
@@ -89,14 +89,15 @@ export default class AvField extends Component {
       size={size}
       disabled={disabled}
       readOnly={readOnly}
+      aria-describedby = {`${id}-error`}
```

Comment:
Let's also make this conditional using the same logic for `AvFeedback`. We don't want to point to an element that doesn't exist. 

Also, please remove the spaces around the `=` to match the other props.